modules/restore-replace-crashlooping-etcd-member.adoc

```diff
@@ -44,6 +44,11 @@ sh-4.2# chroot /host
 +
 [source,terminal]
 ----
+sh-4.2# mkdir /var/lib/etcd-backup
```

Comment:
We probably don't need this, if we modify the 1c. step to move to `/tmp`

```
sh-4.2# mv /etc/kubernetes/manifests/etcd-pod.yaml /tmp
```

Reply:
Yep, I will look into this in a separate PR so that we can make sure that moving there won't impact anything else